R/pkg_ref_cache_r_cmd_check.R

```diff
@@ -0,0 +1,21 @@
+#' Run R CMD check and capture the results
+#'
+#' @inheritParams pkg_ref_cache
+#' @family package reference cache
+#'
+pkg_ref_cache.r_cmd_check <- function (x, ...)
+{
+  UseMethod("pkg_ref_cache.r_cmd_check")
+}
+
+pkg_ref_cache.r_cmd_check.default <- function (x, ...) {
+  return(NA)
+}
+
+
+#' @importFrom devtools check
+pkg_ref_cache.r_cmd_check.pkg_source <- function(x, ...){
+  check_results <- check(x$path)
```

Comment:
I know it's not absolutely necessary, but to keep consistency can we call `devtools::check` just to be explicit to new developers.

```suggestion
  check_results <- devtools::check(x$path)
```